### DISCUSSION TITLE
关于 - ek1ng's Blog

### DISCUSSION THREAD

**Discussion by giscus** [ek1ng/blog#11] [Announcements]
# 关于 - ek1ng's Blog

foucs on security and web development - ek1ng - ek1ng's Blog

https://ek1ng.com/about/

**Comment by ek1ng** [comment]
由于[很多hexo主题移除了valine评论系统](hexo-theme-next#188#issuecomment-766578906)，小站使用的volantis在新版本中也不再支持valine了，所以我把评论系统更换成giscus啦，欢迎大火评论呀

**Reply by lanbinshijie** [reply]
加油加油冲冲冲！

**Comment by rantrism** [comment]
您好～我是腾讯云开发者社区运营，关注了您分享的技术文章，觉得内容很棒，我们诚挚邀请您加入腾讯云自媒体分享计划。完整福利和申请地址请见：https://cloud.tencent.com/developer/support-plan
作者申请此计划后将作者的文章进行搬迁同步到社区的专栏下，你只需要简单填写一下表单申请即可，我们会给作者提供包括流量、云服务器等，另外还有些周边礼物。

**Comment by t0hka1** [comment]
我们注意到您已经近3周没有登入PornHub了，请允许我们问候一声您是否一切如常。从您上次访问我们以后，我们已经更新了很多您喜欢的男同性恋电影。 希望能很快再见到你。
